src/app/components/privacy-policy/privacy-policy.component.spec.ts

```diff
@@ -37,10 +41,23 @@ describe('PrivacyPolicyComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(PrivacyPolicyComponent);
     component = fixture.componentInstance;
+    de = fixture.debugElement;
     fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should have equal number of section-title elements as nav elements', () => {
+    expect(component).toBeTruthy();
```

Comment:
Already present in line 49.

Reply:
> We can also compare the title of sidebar with title of right side (.privacy-section-title).

yeah, but the thing is not every sidebar title is same as section title.

Reply:
If we make it same then any issue or will it change the meaning ?

Reply:
No, it will not change the meaning
Should i do that?

Reply:
Yes!